main.js

```diff
@@ -73,18 +74,30 @@ function getDish(currentDish) {
 var favoriteDish = [];
```

Comment:
I would put this near to your other variables; try and group like things together.